org.scala-ide.sdt.core/src/org/scalaide/core/compiler/DeclarationPrinter.scala

```diff
@@ -78,13 +78,21 @@ abstract class DeclarationPrinter extends HasLogger {
       else nonAccess + " " + access
     }
 
-    val name = if (sym.isConstructor) sym.owner.decodedName else sym.nameString
-    compose(
-      // TODO: Annotations
-      flagString(sym, flagMask & sym.flags),
-      if (showKind) sym.keyString else "",
-      sym.varianceString + name + infoString(seenAs) // don't force the symbol
-      )
+    val gsym= sym.getterIn(sym.owner)
+    def hasGetterSetter(sym:Symbol)= gsym != NoSymbol && sym.setterIn(sym.owner) != NoSymbol
+
+    if(hasGetterSetter(sym)){
+      compose("", "var",sym.localName.toString.trim+": "+gsym.tpe.resultType)
```

Comment:
I don't think this is working very well, since modifiers are eluded and hardcoded to `""`. What about `private/protected` vars?. Also, the type should be printed using the same calls as in the original code. I think the proper way would be to use the getter symbol for all of these. For an example about the type:

```
trait Foo[T] {
  var x: T = _
}

class C extends Foo[Int] {
  prinntl(x) // x: Int or x: T?
}